README.md

```diff
@@ -33,27 +33,56 @@ Most of this works around our "User" model which can do multiple things
 
 # Start developing
 
-* clone the repo
-* install pipenv
-* pipenv shell
-* pipenv install --dev
-* ./manage.py migrate --skip-checks
-* ./manage.py loaddata memberservices
-* ./manage.py runserver
+Make sure you have proper python installation on your machine
+
+* python 3.7
+* pipenv from here: https://github.com/pypa/pipenv
+
+Then run
+
+```bash
+git clone git@github.com:TampereHacklab/mulysa.git
+cd mulysa
+pipenv sync --dev
+pipenv shell
+./manage.py migrate --skip-checks
+./manage.py loaddata memberservices
+./manage.py runserver
+```
 
 ## To update localizations
 
-* django-admin makemessages -l fi
-* (edit .po files)
-* django-admin compilemessages
+Always start everything by opening the pipenv shell for this project first! (`pipenv shell`)
```

Comment:
..."or by prepending commands with (`pipenv run`) for system users that does not have shell access normally.

Reply:
have you tried the "suggestion" thingy. it makes a "precommit" on your name in this branch and then I can just click on "yes this is good" and your suggestion becomes a part of this pr. wanna try it out :) (saves me some copy pasting ;)

https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request (step 6)